backend/src/v2/compiler/argocompiler/common.go

```diff
@@ -14,7 +14,23 @@
 
 package argocompiler
 
-import k8score "k8s.io/api/core/v1"
+import (
+	"fmt"
+	wfapi "github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
+	k8score "k8s.io/api/core/v1"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const (
+	DefaultMLPipelineServiceHost     = "ml-pipeline.kubeflow.svc.cluster.local"
+	DefaultMLPipelineServicePortGRPC = "8887"
```

Comment:
nitpick (sorry to be a pain): it's `mlPipelineServiceGRPCPort` over in persistence client (ref: https://github.com/opendatahub-io/data-science-pipelines/blob/master/backend/src/agent/persistence/client/pipeline_client.go#L58)

PortGRPC vs GRPCPort

(GRPCPort reads better to me)

Reply:
(I see that PortGRPC lines up better with the env var below. Feel free to ignore this one.)